.github/styles/Bokeh/WordList.yml

```diff
@@ -0,0 +1,41 @@
+extends: substitution
+message: "Use '%s' instead of '%s' (if possible)."
+level: error
+ignorecase: false
+action:
+  name: replace
+swap:
+  '(?:API Console|dev|developer) key': API key
+  '(?:e-mail|Email|E-mail)': email
+  '(?:kill|terminate|abort)': stop|exit|cancel|end
```

Comment:
Those are not synonyms from technical language perspective. For example, "killing" and "terminating" a process are well defined and distinct terms (e.g. no cleanup vs. normal cleanup). They can't be replaced by "exit", because this term can mean either. I understand this is a suggestion (?) (if possible), but if this is going to be showed all the time, then people will simply ignore all those suggestions (as we do with warnings in other contexts and the wonder why things "suddenly" stop working).

Reply:
Some very good points, thanks! This was based on a premade ruleset from Vale which is in turn based on the Google developer documentation style guide. But customizing those rulesets to Bokeh is the main reason for having this pull request und this discussion! 😀 I'll remove this entry from the ruleset.